.gitignore

```diff
@@ -13,6 +13,7 @@ dlm/
 lib/
 log/
 
+codebase/superdarn/src.lib/tk/idl.tar.gz
```

Comment:
@mts299 Is this supposed to be here? Or is this just for your system since you don't use IDL?

Reply:
system

Reply:
``` suggestion
codebase/superdarn/src.lib/tk/idl.tar.gz
```